backend/db/migrate/20220817011608_create_property_assessments.rb

```diff
@@ -51,7 +53,7 @@ def change
       t.string :bthrm_style3
       t.string :kitchen_type
       t.string :kitchen_style1
-      t.string :kitchen_style
+      t.string :kitchen_style  # is this kitchen_style2?
```

Comment:
I would nuke the comment from this PR and fix/update the field name in a different PR.